test/resque_inline_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class IgnoreExceptionsTest < Minitest::Test
+  def setup
+    Resque.stubs(:redis)
+    Resque.inline = true
+  end
+
+  def teardown
+    Resque.inline = false
+    Resque.unstub(:redis)
+  end
+
+  def test_ignore_exceptions
```

Comment:
We're not actually testing that exceptions are being ignored.

We want to test that `Resque.inline` does what its supposed to do.